pkg/controller/controller_binding.go

```diff
@@ -800,10 +717,10 @@ func (c *controller) reconcileServiceBinding(binding *v1beta1.ServiceBinding) er
 		}
 
 		c.recorder.Event(binding, corev1.EventTypeNormal, successUnboundReason, "This binding was deleted successfully")
-		glog.V(5).Infof(
-			`%s "%s/%s": Successfully deleted ServiceBinding of ServiceInstance "%s/%s" of ClusterServiceClass (K8S: %q ExternalName: %q) at ClusterServiceBroker %v`,
-			typeSB, binding.Namespace, binding.Name, instance.Namespace, instance.Name, serviceClass.Name, serviceClass.Spec.ExternalName, brokerName,
-		)
+		glog.V(5).Info(pcb.Messagef(
+			"Successfully deleted ServiceBinding of %s",
+			pretty.FromServiceInstanceOfClusterServiceClassAtBrokerName(instance, serviceClass, brokerName),
```

Comment:
I'll be honest, I don't think this degree of information in the log message is especially useful.  I know @kibbles-n-bytes had some thoughts about this.  Do we really think this adds useful information?  I think the salient point in this case is that the binding was deleted.